pylatex/document.py

```diff
@@ -54,6 +54,8 @@ def __init__(self, default_filepath='default_filepath', *,
             Controls the sepparation between the header and the content
         page_numbers: bool
             Adds the ability to add page numbers to the document
+        last_page: bool
+            Determines if the last page package is used (compile twice)
```

Comment:
I think its better to use the `latexmk` command tool to handle this kind of stuff. 
It is the default compiler for pylatex so if you install it it should work fine.

If this is not possible for you I think it would be better to have a new `compiler_runs` argument for `generate_pdf` to specify the number of compilation attempts that are necessary.